test/tests/postprocessors/current_boundary_id/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  issues = '#13953'
+  design = 'Assembly.md'
+  [current_boundary_id]
+    type = CSVDiff
+    input = current_boundary_id.i
+    csvdiff = 'current_boundary_id_out.csv'
+    requirement = "The current boundary id shall be available in objects evaluated on boundaries."
```

Comment:
Are we not using "The system shall..." for requirements, @permcody ? Just checking, because there was some significant amount of work going into SQA, so we don't want to create more work...

Reply:
The system shall is definitely the direction we've received from both the NQA-1 consultant and the INL SQA reps. "The boundary id..." is not a good requirement because boundary id isn't defined in the context of a requirements document. That's an implementation detail.